src/litserve/server.py

```diff
@@ -454,17 +458,15 @@ def __init__(
         self.app.response_queue_id = None
         self.response_queue_id = None
         self.response_buffer = {}
-        # gzip does not play nicely with streaming, see https://github.com/tiangolo/fastapi/discussions/8448
```

Comment:
you can append these middleswares to `self.middlewares` and add it later below in `setup_server` method

Reply:
done